src/App.test.tsx

```diff
@@ -1,8 +1,8 @@
 import { render, screen } from '@testing-library/react';
 import App from './App';
```

Comment:
We should just delete this test file

Reply:
I fixed it up hoping that a stretch goal would be that someone would write a test first if they pushed for TDD

Reply:
Up tp you. Testing this sort of code is pretty tedious and testing library-specific, and there are a bunch of other stretch goals we'd probably get to first